client/cli/src/main.rs

```diff
@@ -375,20 +375,25 @@ async fn main() {
             );
         };
     } else {
+        let module_gens = ModuleGenRegistry::from(vec![
+            DynModuleGen::from(WalletGen),
+            DynModuleGen::from(MintGen),
+            DynModuleGen::from(LightningGen),
+        ]);
```

Comment:
A bit off topic, but couldn't we generate the `decoders` further down from these `module_gens`?

Reply:
They are declared further down, or do you mean right after?

Reply:
Instead of declaring them they could be auto-generated I assume?